### PR TITLE
Removed color abstraction

### PR DESCRIPTION
**Pull request by wakooka**
## Description

Removed the color abstraction and use the paletter colors directly.

## What has been done

All files using colors have been updated to the new color name.

## How it was tested

Locally in storybook

image

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 # Unreleased
 
+- **[BREAKING CHANGE]** Removed color abstraction and only use palette instead
```
Comment (pith) [inline CHANGELOG.md:3]: Are you ready to update Kairos with that breaking change? 😱

Comment (wakooka) [reply CHANGELOG.md:3]: Not yet, I'll prepare the branch before release ;)

**Comment by gabrielagabriel** [comment]
IMHO we should NOT remove "theme" properties. Dark Theme will be **impossible** if we do that

**Comment by wakooka** [comment]
> IMHO we should NOT remove "theme" properties. Dark Theme will be impossible if we do that
I'm only doing what's been decided during the chapter meeting :/
We also said that we were not against having a Theme, but right now the theme we use is a mess

**Pull request merged**